WebHostLib/templates/playerTracker.html

```diff
@@ -0,0 +1,176 @@
+{% block head %}
+    <!--suppress XmlDuplicatedId -->
+<title>{{ player_name }}&apos;s Tracker</title>
```

Comment:
This won't always be grammatically correct. Not a big deal, just pointing it out.